arch/arm/boot/dts/overlays/revpi-core-1-1-overlay.dts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Core 1.1
+ */
+
+#include "revpi-core-overlay.dts"
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/{
+	compatible = "brcm,bcm2711", "brcm,bcm2837";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-core-1.1", "brcm,bcm2711";
```

Comment:
kunbus,revpi-core-1.1 or kunbus,revpi-core-1-1 ? @iluminat23

Reply:
we shouldn't use period, as spec:
_"The compatible string should consist only of lowercase letters, digits and dashes, and should start with a letter."_